docs/Getting Started/Debian/Debian Trixie Root on ZFS.rst

```diff
@@ -0,0 +1,1207 @@
+
+#. Install the tools::
+
+    sudo apt install python3-pip
+
+    pip3 install -r docs/requirements.txt
+
+    # Add ~/.local/bin to your $PATH, e.g. by adding this to ~/.bashrc:
+    PATH=$HOME/.local/bin:$PATH
+
+#. Make your changes.
+
+#. Test::
+
+    cd docs
+    make html
+    sensible-browser _build/html/index.html
+
+#. ``git commit --signoff`` to a branch, ``git push``, and create a pull
+   request. Mention @rlaager.
+
+Encryption
+~~~~~~~~~~
+
+This guide supports three different encryption options: unencrypted, ZFS
+native encryption, and LUKS. With any option, all ZFS features are fully
+available.
+
+Unencrypted does not encrypt anything, of course. With no encryption
+happening, this option naturally has the best performance.
+
+ZFS native encryption encrypts the data and most metadata in the root
+pool. It does not encrypt dataset or snapshot names or properties. The
+boot pool is not encrypted at all, but it only contains the bootloader,
+kernel, and initrd. (Unless you put a password in ``/etc/fstab``, the
+initrd is unlikely to contain sensitive data.) The system cannot boot
+without the passphrase being entered at the console. Performance is
+good. As the encryption happens in ZFS, even if multiple disks (mirror
+or raidz topologies) are used, the data only has to be encrypted once.
+
+LUKS encrypts almost everything. The only unencrypted data is the bootloader,
+kernel, and initrd. The system cannot boot without the passphrase being
+entered at the console. Performance is good, but LUKS sits underneath ZFS, so
+if multiple disks (mirror or raidz topologies) are used, the data has to be
+encrypted once per disk.
+
+Step 1: Prepare The Install Environment
+---------------------------------------
+
+#. Boot the Debian GNU/Linux Live CD. If prompted, login with the username
+   ``user`` and password ``live``. Connect your system to the Internet as
+   appropriate (e.g. join your WiFi network). Open a terminal.
+
+#. Setup and update the repositories::
+
+     sudo vi /etc/apt/sources.list
+
+   .. code-block:: sourceslist
+
+     deb http://deb.debian.org/debian trixie main contrib non-free-firmware
+
+   ::
+
+     sudo apt update
+
+#. Optional: Install and start the OpenSSH server in the Live CD environment:
+
+   If you have a second system, using SSH to access the target system can be
+   convenient::
+
+     sudo apt install --yes openssh-server
+
+     sudo systemctl restart ssh
+
+   **Hint:** You can find your IP address with
+   ``ip addr show scope global | grep inet``. Then, from your main machine,
+   connect with ``ssh user@IP``.
+
+#. Disable automounting:
+
+   If the disk has been used before (with partitions at the same offsets),
+   previous filesystems (e.g. the ESP) will automount if not disabled::
+
+     gsettings set org.gnome.desktop.media-handling automount false
+
+#. Become root::
+
+     sudo -i
+
+#. Install ZFS in the Live CD environment::
```

Comment:
I was using the current live Debian 13.1 ISO with GNOME, and that didn't work. This did work in my case: https://github.com/openzfs/openzfs-docs/blob/master/docs/Getting%20Started/Debian/Debian%20Buster%20Root%20on%20ZFS.rst